apps/openmw/mwmechanics/actors.cpp

```diff
@@ -1130,15 +1128,31 @@ namespace MWMechanics
         }
     }
 
-    Actors::Actors() {
+    Actors::Actors()
+    {
         mTimerDisposeSummonsCorpses = 0.2f; // We should add a delay between summoned creature death and its corpse despawning
+
+        updateProcessingRange();
     }
 
     Actors::~Actors()
     {
         clear();
     }
 
+    float Actors::getProcessingRange() const
+    {
+        return mActorsProcessingRange;
+    }
+
+    void Actors::updateProcessingRange()
+    {
+        float actorsProcessingRange = Settings::Manager::getFloat("actors processing range", "Game");
+        actorsProcessingRange = std::min(actorsProcessingRange, 7168.f);
```

Comment:
Please don't use magic constants ♥

Reply:
It's not really a magic number. It's a sensible upper limit.

Reply:
I suppose we want to say that there should be a named constant.

Reply:
Ho, yes, this is what I meant: "Please don't use magic numbers, a named constant would be better, thank you :) "